modules/core/src/main/java/org/apache/ignite/internal/util/HashCalculator.java

```diff
@@ -94,6 +96,15 @@ public void appendNull() {
         appendByte((byte) 0);
     }
 
+    /**
+     * Append boolean to hash calculation.
+     *
+     * @param v Value to update hash.
+     */
+    public void appendBoolean(boolean v) {
+        hash = HashUtils.hash32(ByteUtils.booleanToByte(v), hash);
+    }
```

Comment:
Please note the pattern with other existing methods. We should have something like this:

```java
    public static int hashBoolean(boolean v) {
        return HashUtils.hash32(ByteUtils.booleanToByte(v));
    }

    public void appendBoolean(boolean v) {
        hash = combine(hash, hashBoolean(v));
    }
```

Reply:
Done, thanks.